tests/contract-playground/src/parent_chain/ParentChainContract.sol

```diff
@@ -13,7 +13,9 @@ contract ParentChainContract is Ownable, ReentrancyGuard {
     }
 
     function increment(uint256 newNumber) public {
+        // @nyth:blame(constants-instead-of-literals:2)
```

Comment:
verify_blame_coverage makes sure that there are exactly 2 instances of next line captured

Reply:
Now, there may be other lines that are captured, but since its not commented with `@nyth:blame` it doesn't matter . test still passes. 

so it's more like use this in places that you think are critical to make it in....

Reply:
and you don't have to use this more than you'd like to use it in a codebase.... (basically `@nyth:blame(..)` need not be marked before every single instance that's captured)